lib/websocket_server.rb

```diff
@@ -25,8 +29,10 @@ def initialize
         reads.each do |socket|
           begin
             @pairing[socket].proxy.transmit(writes, @pairing[socket].is_ws)
-          rescue
+          rescue IOError
```

Comment:
@skateman Can you explain why this change in the commit message and PR description?  this feels like a bug fix in the midst of adding a logger.